src/search.js

```diff
@@ -0,0 +1,79 @@
+import axios
+    from "axios";
+
+const searchButton = document.getElementById("searchButton")
+searchButton.addEventListener('click',getCountryDataOnSearch);
+
+const result = document.getElementById("searchResult");
+
+const searchField = document.getElementById("searchcountry");
+searchField.addEventListener("keypress", function (e) {
+    if (e.code === "Enter") {
+        getCountryDataOnSearch();
+    }
+});
+
+async function getCountryDataOnSearch() {
+    try{
+
+        // leeg maken van resultplekje:
+        result.textContent = "";
+
+// aanroepen van de data
+        const searchQuery = document.getElementById("searchcountry").value;
+
+        const response = await axios.get(`https://restcountries.com/v2/name/${searchQuery}`);
+        const countries = response.data[0];
+
+        console.log(countries);
+        //resultaat data specificeren
+        const nameCountry = countries.name;
+        const flagCountry = countries.flags.png;
+        const subareaCountry = countries.subregion;
+        const populationCountry = countries.population;
+        const capitalCountry = countries.capital;
+        let currenciesCountry = "";
+        // currencies ophalen
+        if (countries.currencies.length === 1) {
+            currenciesCountry = countries.currencies[0].name + "'s";
+        }else {
+            for (let i = 0; i < countries.currencies.length ; i++) {
+                currenciesCountry += countries.currencies[i].name;
+                currenciesCountry += "'s and "
+            }
+            // laatste and weghalen
+            currenciesCountry = currenciesCountry.substring(0, currenciesCountry.length - 5);
+        }
```

Comment:
Je zou hier gebruik kunnen maken van de string.join functie zodat je geen if-else hoeft te gebruiken
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join

Reply:
Thanks, dat is inderdaad een heel stuk korter en ook netter!